tools/osx/macosx_bundle.sh

```diff
@@ -11,7 +11,7 @@
 # - GTK_PREFIX
 
 function message {
-    printf '\e[34m-- %s\e[m\n' "$*"
+    printf '\e[35m-- %s\e[m\n' "$*"
```

Comment:
Personally I would not use colors. I have used bold in the past, but I also ended up removing that, as when people copypasta terminal output into a bug report then the formatted text gets garbled.